### PR TITLE
Add io.liri.Calculator

### PR DESCRIPTION
**Pull request by plfiorini**
Calculator with Material Design from Liri

https://liri.io
https://github.com/lirios/calculator

### CODE REVIEW

**Comment by plfiorini** [comment]
This depends on #654

File: io.liri.Calculator.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+branch: stable
+runtime: org.kde.Platform
+runtime-version: 5.11
+sdk: org.kde.Sdk
+base: io.liri.BaseApp
+base-version: stable
+command: liri-calculator
+finish-args:
+- --device=dri
+- --filesystem=home
```
Comment (TingPing) [inline io.liri.Calculator.yaml]: A calculator should not need this.

Comment (plfiorini) [reply io.liri.Calculator.yaml]: This one lets you load/save calculations in a script

Comment (TingPing) [reply io.liri.Calculator.yaml]: Still shouldn't be needed, the Qt FileChooser dialog should be using the portal to save/load files on the host.

Comment (plfiorini) [reply io.liri.Calculator.yaml]: Done

File: io.liri.Calculator.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+base: io.liri.BaseApp
+base-version: "5.11"
+command: liri-calculator
+finish-args:
+- --device=dri
+- --socket=wayland
+- --socket=x11
+- --share=ipc
+- --env=SESSION_MANAGER=
+cleanup:
```
Comment (TingPing) [inline io.liri.Calculator.yaml]: I presume none of these are needed.

Comment (plfiorini) [reply io.liri.Calculator.yaml]: `SESSION_MANAGER` was removed to silence this Qt error message:
`Qt: Session management error: Could not open network socket`

The `--socket=x11` and `--share=ipc` are need to run the app on X11.

Comment (TingPing) [reply io.liri.Calculator.yaml]: I was referring to the `cleanup` list in all of these apps.

Comment (plfiorini) [reply io.liri.Calculator.yaml]: Done

Comment (barthalion) [reply io.liri.Calculator.yaml]: The message is harmless though so I don't see much gain in silencing it.

**Comment by barthalion** [comment]
bot, build io.liri.Calculator

File: io.liri.Calculator.yaml (hunk 2 above)
Comment (plfiorini) [reply io.liri.Calculator.yaml]: Done

**Comment by barthalion** [comment]
bot, build io.liri.Calculator

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/io.liri.Calculator

Thanks!

**Comment by plfiorini** [comment]
Thanks

**Pull request closed without merging**